packages/mobile/src/app/reducers.ts

```diff
@@ -20,11 +20,11 @@ const initialState = {
   numberVerified: false,
   language: null,
   doingBackupFlow: false,
-  doingPinVerification: false,
   analyticsEnabled: true,
   lockWithPinEnabled: false,
   appState: AppState.Active,
   locked: false,
+  requestingAndroidPermission: false,
```

Comment:
Is this definitely just an android issue?

Reply:
Yes, see here: https://reactnative.dev/docs/appstate
`background`: [Android] on another Activity (even if it was launched by your app)